site/gatsby-site/src/pages/incidents/edit.js

```diff
@@ -45,7 +46,12 @@ function EditCitePage(props) {
       });
 
       addToast({
-        message: `Incident ${incidentId} updated successfully.`,
+        message: (
```

Comment:
Looks like this is missing translations?

Reply:
I've added the translation calls and English on both sides. As far as I can tell, `<Trans>` doesn't work inside the toasts for some reason, so in a couple places I had to use two `t()`. I included complete sentences in each one, so I think that should work for any language.

Reply:
Oh yes, it works but needs to be provided with a `i18n` instance. My guess is that it can't reach react's context, but there is a workaround here: https://github.com/responsible-ai-collaborative/aiid/blob/30f6d9a23246d14d7dc89a9ad42eebe3eb0cc7d6/site/gatsby-site/src/pages/apps/submitted.js#L36 please do it that way if possible.

Reply:
@cesarvarela Fixed. Thanks!